### PR TITLE
[Scaffolding] Node package

### LINKED ISSUE #1008: Scaffolding plugin for a new node package
## This is:
- Feature request

## Detailed Description
We need a new scaffolding plugin to quickly setup a skeleton for a new package for node/api. It should link the package with `tsconfig.build.json` and extend `babel.node.js` file (see our repo for examples).

### PR DESCRIPTION
**Pull request by Fsalker**
## Blog Summary
This plugin allows you to quickly scaffold a Node package. 

The package exports a factory of plugins. There is a sample utility plugin ready for you to use.

## Related Issue
Closes #1008

### CODE REVIEW

File: packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (Pavel910) [inline packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx:1]: React inside of a `node` package? :P

File: packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+export default {
+    name: "utility-plugins-remove-vowels",
+    type: "utility-plugins",
+    removeVowels: str => {
+        return str.replace(/[aeiouAEIOU]/g, "");
+    }
+};
```
Comment (Pavel910) [inline packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx:9]: This whole file can just be this:
```
export default () => {
    // TODO: implement your package
}
```

It's just a skeleton for a generic node package, the goal is to have the configs in place, which you did. Let the developer bother with implementation.

**Comment by Fsalker** [comment]
I'll update this PR soon™

File: packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx (hunk 2 above)
Comment (Fsalker) [reply packages/cli-plugin-scaffold-node-package/template/src/plugins/index.tsx]: done

**Pull request merged**